src/DssProxyActionsCharter.sol

```diff
@@ -338,21 +392,25 @@ contract DssProxyActionsCharter is Common {
     }
 
     function wipe(
-        bytes32 ilk,
         address daiJoin,
+        uint256 cdp,
         uint256 wad
     ) external {
+        address owner = CdpRegistryLike(cdpRegistry).owns(cdp);
+        bytes32 ilk = CdpRegistryLike(cdpRegistry).ilks(cdp);
+
         // Joins DAI amount into the vat
-        daiJoin_join(daiJoin, wad);
+        daiJoin_join(daiJoin, owner, wad);
         // Allows charter to access to proxy's DAI balance in the vat
         VatLike(vat).hope(charter);
         // Paybacks debt to the CDP
         _frob(
             ilk,
+            CdpRegistryLike(cdpRegistry).owns(cdp),
```

Comment:
```suggestion
            owner
```
Or is there any kind of stack too deep?

Reply:
No, just missed that.
Taking your siggestion.